### DISCUSSION TITLE
Dependent dropdowns, multi select not clearing on change

### DISCUSSION THREAD

**Discussion by talhav** [filamentphp/filament#3317] [Help]
I have a country (single select) and cities (multiple select) dependent dropdowns.
if I select one country and then select the multiple cities of that country, it is working fine
but then, when I change country again, the old values are still there in cities dropdown, they are not clearing, I have added afterStateUpdated() as well, but no luck in multi select.
single select works fine

here is when I select oman, and selecting these cities
![image](https://user-images.githubusercontent.com/40432154/181994607-f6ba8918-ee66-40d6-99ee-7595320e0f49.png)

But now, when I change the country to Austria, old cities are still selected
![image](https://user-images.githubusercontent.com/40432154/181994638-3c7a3b34-68c1-47c4-914f-4268d26dfc76.png)

and if I clear the country selection, still the old cities are here:
![image](https://user-images.githubusercontent.com/40432154/181994659-2d34ea53-041e-4f3f-b827-0a7c86932740.png)

**Comment by danharrin** [comment]
Please share code
